bcdevexchange/wwwroot/css/style.css

```diff
@@ -164,7 +164,17 @@ h3 {
 
 .header-area .main-header-area .main-menu ul li a {
   color: #fff;
+  opacity: 0.5;
+  font-weight: 400;
+  letter-spacing: 0.3px;

```

Comment:
Is 0.3 pixels actually visible?

Reply:
Fair question - it actually is visible, just makes the words slightly less crunched.